Main.java

```diff
@@ -0,0 +1,66 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        if (args.length != 2) {
+            System.out.println("File path and search word cannot be empty");
+            return;
+        }
+        String textFilePath = args[0];
+        String searchKeyword = args[1];
+        File file = new File(textFilePath);
+        System.out.println("Processing...");
+        validateFile(file);
+        if (validateFile(file)) {
+            wordCount(file, searchKeyword);
+        }
+    }
+
+    private static boolean validateFile(File file) {
+
+        if (!file.exists()) {
+            System.out.println("File does not exist");
+
+        } else if (!(file.getName().endsWith(".txt") || file.getName().endsWith(".json"))) {

```

Comment:
Use static final constants instead of hardcoding .txt and .json

Reply:
Sure. Will do that.